numpy/linalg/lapack_lite/make_lite.py

```diff
@@ -232,6 +240,8 @@ def getWrappedRoutineNames(wrapped_routines_file):
 types = {'blas', 'lapack', 'd_lapack', 's_lapack', 'z_lapack', 'c_lapack', 'config'}
 
 def dumpRoutineNames(library, output_dir):
+    if not PY2:
+        os.makedirs(output_dir, exist_ok=True)
```

Comment:
Not sure it's worth doing this if we only do it on py3, nor am I sure that creating the output directory is in-scope for what the build script should do.

Reply:
`exist_ok` doesn't exist on PY2, and I don't enjoy making up for its shortcomings.